src/components/JaegerIntegration/JaegerResults/SpanDetail.tsx

```diff
@@ -56,10 +56,10 @@ export class SpanDetailC extends React.Component<SpanDetailProps, SpanDetailStat
                     </GridItem>
                     <GridItem span={4} />
                     <GridItem span={2}>
-                      <Tooltip content={<>View traces of service {this.props.spans[0].process.serviceName}</>}>
+                      <Tooltip content={<>View traces of app {this.props.spans[0].process.serviceName}</>}>
```

Comment:
I think we tend to not use "app" as a word in general (correct me if I'm wrong), we display the full word "application" instead.
Perhaps here we can just write "View traces of xxx".

Reply:
right and changed